gen/src/generate/ucd/normal.rs

```diff
@@ -147,10 +191,11 @@ impl CanonicalCompositionData {
             }),
         )
     }
-}
 
-impl<'a, 'b> From<&'a CanonicalDecompositionData<'b>> for CanonicalCompositionData {
```

Comment:
What happened here? I'm not sure why this `From` was needed, but also don't understand why to have a non-`From` impl of `from`. Are we de-generalizing the impl? (Too bad we don't have a test for this expection...)